x/oracle/client/cli/tx.go

```diff
@@ -61,3 +63,70 @@ func GetCmdDelegateFeedConsent() *cobra.Command {
 
 	return cmd
 }
+
+// GetCmdAggregateExchangeRatePrevote creates a Cobra command to generate or
+// broadcast a transaction with a MsgAggregateExchangeRatePrevote message.
+func GetCmdAggregateExchangeRatePrevote() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "exchange-rate-prevote [hash] [feeder]",
```

Comment:
`feeder` is `--from`. Do we need it? ref: https://github.com/umee-network/umee/issues/1205

Reply:
Otherwise make sure that `--from == args[1]`